### PR TITLE
feat(warn-instance-number): add warning on instance number

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-735

2023-07-19-163323_1495x1097_scrot
2023-07-19-143338_1665x1216_scrot
2023-07-19-143411_1600x1316_scrot

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

File: libs/pages/application/src/lib/ui/page-settings-healthchecks/page-settings-healthchecks.tsx [hunk 1]
```diff
@@ -22,32 +25,36 @@ export function PageSettingsHealthchecks({
...
-            type={BannerBoxEnum.WARNING}
             message={
-              <>
-                Please configure a port before using any health check types, except for the EXEC type.{' '}
+              <span>
+                Your service is configured to run with a minimum of one instance, setting the health checks will not
+                ensure the service high availability during a cluster upgrade. Have a look at your{' '}
                 <Link
-                  className="link !block text-accent2-500 mt-1"
+                  className="link text-accent2-500 mt-1"
```
Comment (RemiBonnet) [inline libs/pages/application/src/lib/ui/page-settings-healthchecks/page-settings-healthchecks.tsx:49]: I think you can remove the `mt-1` to remove the UI offset

**Review by RemiBonnet — COMMENTED** [review]
Thanks and well done @ctjhoa 🥇 

LGTM, just one UI issue with a strange space between the line with the link, I think you just need to remove the `mt-1`

File: libs/pages/application/src/lib/ui/page-settings-healthchecks/page-settings-healthchecks.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/pages/application/src/lib/ui/page-settings-healthchecks/page-settings-healthchecks.tsx]: Fixed
https://github.com/Qovery/console/pull/680/commits/6c3f7f6f25ff86ed40fc4362f42a5ea35041749c

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**